cypress/utils/utils.ts

```diff
@@ -1456,28 +1456,28 @@ export function selectWithinModal(selector: string): void {
     clickWithin(modal, selector);
 }
 
-export function clickWithin(parent, selector: string): void {
+export function clickWithin(parent, selector: string, isForced = false, log = false): void {
     cy.get(parent, { timeout: 30 * SEC })
         .eq(0)
         .within(() => {
-            click(selector);
+            click(selector, isForced, log);
         });
 }
 
 //function to select checkboxes
-export function selectCheckBox(selector: string): void {
+export function selectCheckBox(selector: string, isForced = false, log = false): void {
```

Comment:
can select and unselect chechbox method be combined ?

Reply:
No, unfortunately I couldn't make it work in the past. Anyway, let it be another PR.